src/uikit.cs

```diff
@@ -127,6 +143,43 @@ interface NSAttributedStringAttachmentConveniences {
 	}
 #endif
 
+	[NoWatch, NoTV, iOS (10,0)]
+	[DisableDefaultCtor]
+	[Abstract] // abstract class that should not be used directly
+	[BaseType (typeof (NSObject))]
+	interface UIFeedbackGenerator {
+
+		[Export ("prepare")]
+		void Prepare ();
+	}
+
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(UIFeedbackGenerator))]
+	interface UIImpactFeedbackGenerator {
+
+		[Export ("initWithStyle:")]
+		IntPtr Constructor (UIImpactFeedbackStyle style);
+
+		[Export ("impactOccurred")]
+		void ImpactOccurred ();
+	}
+
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(UIFeedbackGenerator))]
+	interface UINotificationFeedbackGenerator {
```

Comment:
this likely needs to inline the base .ctor - we only warn for designated initializer (but I guess it should be one)